src/main/java/com/salesforce/dynamodbv2/mt/repo/MtTableDescriptionRepo.java

```diff
@@ -9,18 +9,41 @@
 
 import com.amazonaws.services.dynamodbv2.model.CreateTableRequest;
 import com.amazonaws.services.dynamodbv2.model.TableDescription;
+import com.salesforce.dynamodbv2.mt.mappers.MtAmazonDynamoDb.TenantTable;
+import java.util.Map;
 
 /**
- * TODO: write Javadoc.
+ * Interface for managing table metadata of multi-tenant virtual tables within mt-dynamo.
  *
  * @author msgroi
  */
 public interface MtTableDescriptionRepo {
 
+    /**
+     * Create a multi-tenant (virtual) table with specs defined in @param createTableRequest under the given
+     * multi-tenant contexts' namespace.
```

Comment:
assuming this refers to just one context, replace "contexts'" with "context's"
also, does "@param" just show up as "@param" in the Javadoc?